### PR TITLE
Convert Training Jobs to Pretrained Model Jobs

### PR DESCRIPTION
**Pull request by Lucaszw**
depends on #896 . Convert existing training jobs into pretrained models.

### CODE REVIEW

**Comment by Lucaszw** [comment]
Re-based to avoid conflicts with other #896

**Comment by Lucaszw** [comment]
@lukeyeager , Re-based to avoid conflicts with last merge

File: digits/model/views.py [hunk 1]
```diff
@@ -177,6 +177,53 @@ def visualize_lr():
 
     return json.dumps({'data': {'columns': datalist}})
 
+
+@blueprint.route('/<job_id>/to_pretrained',
+        methods=['GET', 'POST'],
+        defaults={'extension': 'tar.gz'})
+@blueprint.route('/<job_id>/to_pretrained.<extension>',
+        methods=['GET', 'POST'])
+def to_pretrained(job_id, extension):
```
Comment (lukeyeager) [inline digits/model/views.py]: Don't need extension here. Copy+Paste error?

File: digits/static/css/style.css [hunk 2]
```diff
@@ -86,8 +86,8 @@ path.c3-line {
 }
 
 /* Hide all data transformation and augmentation by default */
-.data_transformation_input{ display:none; } 
-.data_augmentation_input{ display:none; } 
+.data_transformation_input{ display:none; }
+.data_augmentation_input{ display:none; }
```
Comment (lukeyeager) [inline digits/static/css/style.css]: One more unnecessary whitespace change here

**Comment by Lucaszw** [comment]
@lukeyeager Thanks! Added your changes :)

File: digits/static/js/home_app.js [hunk 3]
```diff
@@ -531,15 +531,16 @@
     app.controller('pretrained_models_controller', function($scope, $localStorage, $controller) {
         $controller('job_controller', {$scope: $scope});
         $scope.title = 'Models';
-        $scope.storage = $localStorage.$default({
+        $scope.fields = {
```
Comment (lukeyeager) [inline digits/static/js/home_app.js]: :+1:

**Comment by lukeyeager** [comment]
Good stuff, thanks! I'll merge after Travis passes.
